### ISSUE TITLE
Add syntax highlighting

### ISSUE THREAD

**Issue by flexagoon** [soupslurpr/BeauTyXT#156]
It would be awesome to have syntax highlighting for various code languages and other plaintext file types (eg. json)

**Comment by Tachi107** [comment]
Yeah, I sometimes have a quick look at patches which I receive via email, and having syntax highlighting would make viewing them way easier!

Btw, beautyxt is really nice, congrats!

**Comment by soupslurpr** [comment]
> [!NOTE]
> gpt-6-astra (max reasoning effort), responding on behalf of soupslurpr.

[BeauTyXT Reimagined (84)](https://github.com/soupslurpr/BeauTyXT/releases/tag/84) renders Markdown code blocks, but it does not add language-aware syntax highlighting for source files, JSON, patches, or code blocks.

Syntax highlighting can make code and structured text easier to scan, but it isn't part of BeauTyXT's current priorities. Closing this as not planned.

**Comment by Tachi107** [comment]
Having an LLM chatbot decide the _direction_ of a project, choosing what 
is planned and what is not, is... weird?

**Comment by soupslurpr** [comment]
> Having an LLM chatbot decide the _direction_ of a project, choosing what
> is planned and what is not, is... weird?

To be clear, that wasn't a fully automated response. I was the one who decided that. The response was initially just an update for the status in Reimagined (which was fully automated as I wanted to go through each issue and post an update), but the original was edited rather than making a new response for the decision that it isn't in BeauTyXT's current priorities.

**Comment by Tachi107** [comment]
Got it, thanks for the reply :)

It wasn't obvious from the message I got via email.
